### PR TITLE
chore: update to 4.30

### PR DESCRIPTION
**Pull request by andygup**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,301 +1,119 @@
...
+The `next` version of 4.30 is now available. Planned release date is June 2024.
 
-![Current build version](https://img.shields.io/npm/v/arcgis-js-api/next?label=Current%20build)
-
-## Performance updates
-
-The [heatmapRendererCreator.createRenderer()](https://developers.arcgis.com/javascript/latest/api-reference/esri-smartMapping-renderers-heatmap.html#createRenderer) method for generating a [HeatmapRenderer](https://developers.arcgis.com/javascript/latest/api-reference/esri-renderers-HeatmapRenderer.html) is 100x faster than it was in version 4.28.
+![Current build version](https://img.shields.io/npm/v/@arcgis/core/next?label=Current%20build)
 
 ## Layer updates
```
Comment (lboyd93) [inline CHANGELOG.md:7]: I would probably just remove this or put TBD if it's going to be empty

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,301 +1,119 @@
...
-
-### VectorTileLayer hitTest
-
-The [hitTest()](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-MapView.html#hitTest) method on the MapView now returns features belonging to a [VectorTileLayer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-VectorTileLayer.html) that intersect the hitTest point. This provides access to the attributes of those features, which can be useful for various purposes such as displaying additional information or performing further analysis.
-
-### MapView hitTest respects drawing order
-
-When performing a [`hitTest()`](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-MapView.html#hitTest) on the MapView, the drawing order of the features in the layer is now respected. The returned features will adhere to the default drawing order of the data or a specific order if [`orderBy`](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-FeatureLayer.html#orderBy) is specified on the layer. This is a significant improvement from previous versions where the features were displayed based on the order they were returned from the server.
-
 ## Widget Updates
```
Comment (lboyd93) [inline CHANGELOG.md:9]: Same for this one: I would probably just remove this or put TBD if it's going to be empty

**Review by lboyd93 — APPROVED** [review]
LGTM! Left minor suggestion 😃

File: CHANGELOG.md (hunk 1 above)
Comment (andygup) [reply CHANGELOG.md:7]: Thanks, I added TBD.

File: CHANGELOG.md (hunk 2 above)
Comment (andygup) [reply CHANGELOG.md:11]: Ditto, and thanks!

**Review by annelfitz — APPROVED** [review]

**Pull request merged**